### PR TITLE
Inline Series.astype documentation

### PR DESCRIPTION
**Pull request by HyukjinKwon**

### CODE REVIEW

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -258,6 +258,36 @@ def spark_type(self):
...
+        numpy.ndarray.astype : Cast a numpy array to a specified type.
+
+        Examples
+        --------
+        >>> ser = ks.Series([1, 2], dtype='int32')
+        >>> ser
+        0    1
+        1    2
+        Name: 0, dtype: int32
+        >>> ser.astype('int64')
```
Comment (rxin) [inline databricks/koalas/series.py]: add a new line here ...

in general i think we should separate doctests with blank lines to make them easier to read

File: databricks/koalas/series.py [hunk 2]
```diff
@@ -258,6 +258,36 @@ def spark_type(self):
...
+            the same type.
+
+        Returns
+        -------
+        casted : same type as caller
+
+        See Also
+        --------
+        to_datetime : Convert argument to datetime.
+        numpy.ndarray.astype : Cast a numpy array to a specified type.
```
Comment (rxin) [inline databricks/koalas/series.py]: remove this

File: databricks/koalas/series.py (hunk 1 above)
Comment (HyukjinKwon) [reply databricks/koalas/series.py]: okie

**Comment by rxin** [comment]
LGTM other than the two minor comments .... and this reminds me, we should implement DataFrame.astype as well.

**Pull request merged**